### DISCUSSION TITLE
Ent and Atlas

### DISCUSSION THREAD

**Discussion by ryanobjc** [ent/ent#4277] [General]
I was somewhat interested in ent, but reading it's docs there is are several upsell to the paid atlas service in the "open source" library documentation.  For example in: https://entgo.io/blog/2023/08/13/visualize-with-atlas/  which appears as an iframe of the docs site (fully embedded), it makes a reference to Atlas Cloud as being a useful thing.

What is going on here?  Is Ent a useful project if you don't adopt atlas?

**Comment by a8m** [comment]
> Is Ent a useful project if you don't adopt atlas?

Yes, you can use Ent without Atlas (or Atlas Cloud). Visualization is not an Ent feature, it's an Atlas feature.
